src/rgw/rgw_main.cc

```diff
@@ -1153,6 +1164,10 @@ int main(int argc, const char **argv)
   register_async_signal_handler(SIGUSR1, handle_sigterm);
   sighandler_alrm = signal(SIGALRM, godown_alarm);
 
+  if ( rgw_rate_limit_init() < 0 ) {
+    dout(0) << "Failed to init rate limiter" << dendl;
```

Comment:
Is continuing if we are unable to init the rate limiter the right thing to do?  We should not startup, innit?